contributing.md

```diff
@@ -155,4 +155,4 @@ Once you've made your changes and tested that it works locally, run the tests us
 
 ## Debugging
 
-When debugging, it can be useful to place a console log of the error in the the try catch block of `./cli/src/commands/create-expo-stack.ts`. This should give you a hint as to what is going wrong.
\ No newline at end of file
+When debugging, it can be useful to place a console log of the error in the the try catch block of `./cli/src/commands/create-expo-stack.ts`. This should give you a hint as to what is going wrong.
```

Comment:
```suggestion
When debugging, it can be useful to place a console log of the error in the try/catch block of `./cli/src/commands/create-expo-stack.ts`. This should give you a hint as to what is going wrong.
```